api/app/controllers/spree/api/v2/platform/resource_controller.rb

```diff
@@ -61,7 +61,7 @@ def collection
           def scope
             return super if spree_current_user.present?
 
-            model_class.includes(scope_includes)
+            super(skip_cancancan: true)
```

Comment:
Why are we skipping authorization when there is no current user? When does this case happen?

Reply:
@mad-eel this is the platform API part - we're already authorizing app client/credentials here: https://github.com/spree/spree/blob/master/api/app/controllers/spree/api/v2/platform/resource_controller.rb#L10

Reply:
we have 2 modes of using this API:

1. Apps - authorized by doorkeeper app token
2. Apps on behalf of users - authorized by doorkeeper app token + doorkeeper user token